pkg/kv/util/weak_owner.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/kv/util";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.kv;
+
+// message data model for weak ownership
+message WeakOwnership {
+    string owner = 1;
+    google.protobuf.Timestamp expires = 2;
+    string comment = 3;
```

Comment:
Suppose a system is behaving really strangely.  I can look at its ownership partition and determine which operation owns it -- that's what I put [here](https://github.com/treeverse/lakeFS/pull/8268/files#diff-3cd676feda5aeb7dfa84f20908a57e475aae2b022bddd9395297307f0b178169R208) when I took ownership.  I call it "comment" rather than something else because I don't want to commit to any format of what exactly goes in there.